src/scene/immediate.js

```diff
@@ -134,29 +134,29 @@ Object.assign(pc.Application.prototype, function () {
     }
 
     function _addLines(position, color, options) {
-        if (options.layer === undefined) options.layer = this.scene.layers.getLayerById(pc.LAYERID_IMMEDIATE);
-        if (options.depthTest === undefined) options.depthTest = true;
+        var layer = (options && options.layer) ? options.layer : this.scene.layers.getLayerById(pc.LAYERID_IMMEDIATE);
```

Comment:
What if I pass `pc.LAYERID_WORLD` which is set to 0?  
https://github.com/playcanvas/engine/blob/master/src/scene/scene.js#L150

Reply:
that should be ok .. you pass in Layer object, not id. That's how this is called:

     * var worldLayer = app.scene.layers.getLayerById(pc.LAYERID_WORLD);
     * app.renderLine(start, end, color, {
     *     layer: worldLayer
     * });

Reply:
Oh yes, you're right. 👍